client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -114,7 +113,7 @@ export default React.createClass( {
 			ref: name,
 			additionalClasses: 'checkout-field',
 			value: formState.getFieldValue( this.state.form, name ),
-			invalid: formState.isFieldInvalid( this.state.form, name ),
+			isError: formState.isFieldInvalid( this.state.form, name ),
```

Comment:
You are changing the prop names from `invalid` to `isError`; are the rest of the prop names are `isError` already? If they're different they might create more confusion.